builders/build/local/adding-built-in-pallet.md

```diff
@@ -35,19 +50,22 @@ In the following example, the popular Substrate module `pallet-assets` is added
 
 Every package contains a manifest file named `Cargo.toml` stating, among other things, all the dependencies the package relies on, and the ContainerChain runtime is no exception.
 
-Therefore, the first step, is to declare the dependency and make it available to the runtime, open the `Cargo.toml` file located in the folder `runtime` with a text editor and add the module, referencing the code in the official repository of the Polkadot SDK:
+Therefore, the first step, is to declare the dependency and make it available to the runtime, open the `Cargo.toml` file located in the folder `container-chains/templates/frontier/runtime` with a text editor and add the module, referencing the code in the Polkadot SDK:
 
 ```toml
 [dependencies]
 ...
 pallet-assets = { 
-   git = "https://github.com/paritytech/polkadot-sdk", 
-   branch = "master", 
+   git = "https://github.com/moondance-labs/polkadot-sdk", 
+   branch = "tanssi-polkadot-v1.1.0", 
```

Comment:
Would it make sense to use variables here in the `variables.yaml` file?

Reply:
Absolutely. I'll add it here, and start another PR to replace the references to the 0.9.43 branch